lib/hal/video.c

```diff
@@ -225,12 +227,11 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 	}
 	if (refresh == 0)
 	{
-		if(dwEnc & 0x00400000)
+		if(is_pal)
 		{
+			refresh = (dwEnc & 0x00400000) ? 60 : 50;
```

Comment:
How do you know what magic number should be used? I fail to find adequate documentation. :/

Reply:
This constant already existed in the code. I do not intend to change it for this PR as this PR focuses on fixing a systematic bug, rather than improving code quality overall.

However, this is [documented in XboxDevWiki](https://xboxdevwiki.net/EEPROM) but also in [my xbox-tools](https://github.com/JayFoxRox/xbox-tools/blob/master/python-scripts/get_av_settings.py#L59). Note that all of this documentation is somewhat vague.
It probably originated from XDK related sources.
I think it's reasonable to believe that there's no real PAL-M and that the 50Hz flag at 0x00800000 is unused or only used on some units.

*Just to avoid follow-up questions after bringing up EEPROM information: I chose to respect the PAL-I video standard flag, instead of EEPROM region code. That is because that's what the MS dashboard seems to check for when handling visibility of the settings (see `(theConfig.GetAVRegion() == "PAL_I")` checks in C:/xboxdashdata.*/default.xip: settings3.xap).*

Reply:
Thanks for the clarification! can be resolved.